biggerquery/deploy.py

```diff
@@ -63,8 +67,8 @@ def blob_URI(blob):
     return f"gs://{blob.bucket.name}/{blob.name}"
 
 
-def upload_DAGs_folder(workdir: str, bucket):
-    dags_dir_path = get_dags_output_dir(workdir)
+def upload_DAGs_folder(dags_dir: str, bucket):
```

Comment:
maybe dags instead of DAGs?

Reply:
Generally, we use CAPS for acronyms in writing as a clear signal that the word is an acronym. What if this rule collide with the Python rule:  "function name should be lowercase" ?

```
get_uri()
```

or

```
get_URI()
```